### ISSUE TITLE
nsqd: a way to configure logging verbosity

### ISSUE THREAD

**Issue by jchauncey** [nsqio/nsq#798]
I have a lot of messages like this - `[nsqd] 2016/10/14 06:12:02.995304 200 GET /stats?format=json (10.2.78.72:34186) 32.394µs` and they are not particularly useful to me.

**Comment by mreiferson** [comment]
Unfortunately no, there are no logging verbosity options.

**Comment by freeseacher** [comment]
+100500 for that. 
My logs full of mostly debug log output. 
i am experimenting with such solution
```
# cat /etc/systemd/system/nsqd.service
[Unit]
Description=Realtime distributed messaging (nsqd)
After=network.target

[Service]
ExecStart=/bin/sh -c '/bin/nsqd -config /etc/nsq/nsqd.conf 2>&1| egrep -v "sending heartbeat|200 GET"'
Restart=always
User=nsq
Group=nsq

[Install]
WantedBy=multi-user.target
```
It works exactly like looks.

**Comment by mreiferson** [comment]
will be fixed in #892, closing
